### DISCUSSION TITLE
Force language

### DISCUSSION THREAD

**Discussion by tivoo** [piitaya/lovelace-mushroom#301] [Q&A]
I'd like my Mushroom cards to be displayed in nl language, without changing the system language. How do I force nl as a language without changing my whole system from en to nl? Thanks!

**Comment by piitaya** [comment]
Hum, why do you want such a thing ? 😅

**Reply by tivoo** [reply]
Because I have other users that use the dashboard and want it in Dutch, but I prefer my UI in English since I write code in English and most of my entities' and devices' names are in English. Would it be easy to change the src code or will that not work?

**Reply by piitaya** [reply]
The language is scoped to user account. Each user can have a different languages.
